src/shared/components/lazyMobXTable/LazyMobXTable.tsx

```diff
@@ -546,6 +551,15 @@ export default class LazyMobXTable<T> extends React.Component<LazyMobXTableProps
         }
     }
 
+    private getButtonControls() {
```

Comment:
maybe rename to `getShowMoreButtonControls` or `getShowMoreButton` and component accordingly